Vereinsstatuten.md

```diff
@@ -0,0 +1,128 @@
+# Statuten des Vereins Rust Basel 
+
+> Version 0.1
+
+## Name und Sitz
+
+Unter dem Namen Rust Basel besteht ein Verein, der den vorliegenden Statuten und den Vorschriften in Art. 60 ff. ZGB untersteht.
+Sitz des Vereins ist Basel Stadt.
+
+## Zweck
+
+Unser Verein hat es sich zur Aufgabe gemacht, die Rust Community im Raum Basel zu fördern und eine Plattform für den Austausch unter technikaffinen Menschen zu bieten. Über diesen Verein sollen Treffen für eine Austausch organisiert und gefördert werden.
+Der Verein ist politisch und konfessionell neutral.
+Einer Umwandlung des Vereinszwecks müssen alle Vereinsmitglieder zustimmen.
+
+## Mitglieder
+
+Mitglieder des Vereins können natürliche und juristische Personen sowie Personengesellschaften sein, welche den Zweck des Vereins anerkennen und fördern.
+Der Vorstand entscheidet über die Aufnahme von Mitgliedern nach schriftlich eingereichtem Aufnahmegesuch. Der Entscheid des Vorstandes ist endgültig. Ein ablehnender Entscheid muss nicht begründet werden.
+
+## Mitgliederbeitrag
+
+Der Mitgliederbeitrag wird von der Vereinsversammlung jährlich festgelegt. Er beträgt maximal CHF______ pro Jahr.
+Mitglieder haben für das Kalenderjahr, in welchem ihre Aufnahme erfolgt bzw. ihre Mitgliedschaft erlischt, den anteilsmässigen Mitgliederbeitrag zu entrichten.
+
+
+## Erlöschen der Mitgliedschaft
+
+### Erlöschensgründe
+
+Die Mitgliedschaft erlischt durch
+    • Austritt
+    • Ausschluss
+    • Tod bei natürlichen Personen bzw. Verlust der Rechtsfähigkeit bei juristischen Personen.
+
+### Austritt
+
+Der Austritt kann unter Einhaltung einer Frist von einem Monat auf das Ende des Kalenderjahrs schriftlich gegenüber dem Vorstand erklärt werden.
+Variante: Der Austritt kann jederzeit schriftlich gegenüber dem Vorstand erklärt werden und tritt sofort in Kraft.
```

Comment:
Man muss sich für eines entscheiden in der Variante - ich wäre für für zweiteres.